vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -32,6 +32,14 @@
 
 #define VIC_MPI_ROOT 0
 
+/******************************************************************************
+ *  * @brief   This structure stores netCDF file name and corresponding nc_id
+ *   *****************************************************************************/
+typedef struct {
+    char    nc_file[MAXSTRING];
```

Comment:
Since it is a `nameid` struct, I would call use `nc_filename` rather than `nc_file`. The latter is ambiguous, since I would have expect `nc_file` to be of type `FILE *` rather than `char *`